BlitzBoard.xcodeproj/project.pbxproj

```diff
@@ -10,6 +10,8 @@
 		450CA0AE2BF5E9E0004FF11B /* SDWebImage in Frameworks */ = {isa = PBXBuildFile; productRef = 450CA0AD2BF5E9E0004FF11B /* SDWebImage */; };
 		450CA0B02BF5E9E0004FF11B /* SDWebImageMapKit in Frameworks */ = {isa = PBXBuildFile; productRef = 450CA0AF2BF5E9E0004FF11B /* SDWebImageMapKit */; };
```

Comment:
Why are there 41 files touched?
Please reas through the standards documents we have put together and the many git lessons we did. We have said quite a few times that PR's should be small (between 1-15) files. Future PR's with more that 10-15 files will be denied on the spot..

Reply:
I apologise for the large file, it requested that I fix up my UI in my HomeScreen feature in this PR before merging. I ran into a few issues during my development process which required me to make adjustments in other files. I won't create a large file again.